pkg/storage/mem/mem.go

```diff
@@ -30,6 +30,6 @@ func NewStorage() (storage.BackendConnector, error) {
 	}
 
 	s := fs.NewStorage(tmpDir, memFs)
-	memStorage = storage.NoOpBackendConnector(s)
+	memStorage = storage.NoOpBackend(s)
```

Comment:
Do we need to wrap the FS storage in a NoOpBackend? FS storage should implement the Backend interface.